azure_functions_worker/dispatcher.py

```diff
@@ -322,7 +322,7 @@ async def _handle__worker_status_request(self, request):
             request_id=request.request_id,
             worker_status_response=protos.WorkerStatusResponse())
 
-    def get_function_metadata(self, directory, caller_info):
+    def get_function_metadata(self, function_app_directory, caller_info):
```

Comment:
nit: change to load_function_metadata? get usually returns something back but here is not case

Reply:
Fixed